src/Sarif.UnitTests/TestData/PrereleaseCompatibilityTransformer/Inputs/ComprehensiveToolProperties.sarif

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-11-28",
+  "version": "2.0.0-csd.2.beta.2018-11-28",
+  "runs": [
+    {
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0",
+        "downloadUri": "https://example.com/sensitive_terms.html",
+        "sarifLoggerVersion": "1.5.0",
+        "fileVersion": "1.2.3.4",
+        "fullName": "CodeScanner for EN-US",
+        "language": "en-us",
+        "version": "2.1.0 optimized release for x86",
+        "properties": {
+          "tags": [ "singleTag" ],
+          "propertyKey": "propertyValue"
+        },
+        "globalMessageStrings": {
```

Comment:
>globalMessageStrings [](start = 9, length = 20)

In 11-28, these three properties didn't yet live in `tool`, right? Global message strings and rules metadata lived in `run.resources`, and notification metadata didn't yet exist.